raiden_libs/test/mocks/client.py

```diff
@@ -368,3 +366,8 @@ def get_channel_participant_info(
             field: channel_info[return_fields.index(field)]
             for field in return_fields
         }
+
+    def set_callback(self):
+        """To be called if a transport is used."""
+        assert self.transport is not None
+        self.transport.add_message_callback(self.on_message_event)
```

Comment:
Why not do this in the constructor as before?

Reply:
This wasn't working, the `self.transport.add_message_callback` was producing an error with `transport = None`.

Reply:
But that should have never been called, right?

Reply:
Yupp, was never called. I wanted to make it clear, that this is optional for the mocked_client, that's the reason for the additional method.